views/result_selection.py

```diff
@@ -194,21 +197,34 @@ def select_ts_datasource(self):
             # Little test for checking if there is an id mapping file available
             # If not we check if an .h5 file is available
             # If not we're not going to proceed
-            try:
-                find_id_mapping_file(filename)
-                ds_type = 'netcdf'
-            except IndexError:
+
+            if detect_netcdf_version(filename) == 'netcdf-groundwater':
```

Comment:
how about this:

```python
ds_type = detect_netcdf_version(filename)
```

THis way, you can remove ds_type from the lines below (lines 204, 217)